recipes/metaprob/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "MetaProb" %}
+{% set version = "2" %}
+{% set sha256 = "ada049a1a9fde3ddd9854230af257ee495518a45f0ce2504272f7bcdc060717b" %}
+{% set md5 = "200690e9513bb4725cfe7fd61f8a8a71" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name|lower }}-{{ version }}.tar.gz
+  url: https://bitbucket.org/samu661/metaprob/downloads/{{ name }}_v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  md5: {{ md5 }}
+  patches:
+    - makefile.patch
+
+build:
+  number: 0
+  skip: True  # [ osx ]
+
```

Comment:
I think you should add: `string: boost{{CONDA_BOOST}}_{{PKG_BUILDNUM}}`

@bgruening could you confirm?